### ISSUE TITLE
add plotting

### ISSUE THREAD

**Issue by bbelderbos** [PyBites-Open-Source/git-stats#5]
maybe using plotext?

**Comment by ishan-nahid** [comment]
what should be plotted? 
describe and assign.

**Comment by bbelderbos** [comment]
Good point, we could start with the reporting you see in the Readme, specifically the first "Repo changes (inserts and deletes) per week" one. Does that help?

I am also open to adding more metrics to make this more interesting, any ideas? If so let's open one or more new issues for that.

I wrote this beginner article on plotext the other day btw: https://pybit.es/articles/terminal-plotting-with-plotext/
